docs/DataTypes.md

```diff
@@ -39,11 +39,11 @@ The format is a subset of [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601); th
 with precisely 3 digits of sub-second precision, plus an optional offset specifier of either "Z" or a "+HH" format. 
 Values have the same maximum precision as Java's [OffsetDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/OffsetDateTime.html) class.
 
-Temporal values can be in the range `0001-01-01T00:00:00.000` to `9999-12-31T23:59:59.999`
+DateTimes can be in the range `0001-01-01T00:00:00.000` to `9999-12-31T23:59:59.999`
 that is **_`midnight on the 1st January 0001`_** to **_`1 millisecond to midnight on the 31 December 9999`_**
 
-Temporal values are by default output per the user's locale, adjusted to their time zone.
+DateTimes are by default output per the user's locale, adjusted to their time zone.
 
-### Temporal granularity
+### DateTime granularity
 
-Temporal values currently have granularities derived from the size of their range. Future work ([#141](https://github.com/ScottLogic/datahelix/issues/141)) will make this configurable.
+DateTimes currently have granularities derived from the size of their range. Future work ([#141](https://github.com/ScottLogic/datahelix/issues/141)) will make this configurable.
```

Comment:
This statement should have been removed/updated as part #667, can this be updated here please.

```suggestion
DateTimes currently have a granularity of 1 millisecond. Future work ([#141](https://github.com/ScottLogic/datahelix/issues/141)) will make this configurable.
```